tic-tac-toe/src/app/screens/Login/fields/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export const customInput = props => {
+  const { label, input, type, meta } = props;
```

Comment:
You can pass the attributes directly in the params like this. function customInput({ label, input, type, meta })